src/components/icon/icon.ts

```diff
@@ -11,7 +11,7 @@ import style from './icon.scss?lit&inline';
  */
 @customElement('sbb-icon')
 export class SbbIcon extends LitElement {
```

Comment:
Maybe using a class mixin? it could make sense
https://lit.dev/docs/composition/mixins/

could it be a reusable pattern also for link-button components (menu-action, header-action, etc)?